.github/workflows/main.yml

```diff
@@ -38,10 +38,27 @@ jobs:
       - name: Update bundler
         env:
           RUBY_VERSION: ${{ matrix.ruby }}
+        # See: https://dev.to/galtzo/matrix-ruby-gem-bundler-etc-4kk7
         run: |
           case ${RUBY_VERSION} in
+            1.8|1.9|2.0|2.1|2.2)
```

Comment:
Maybe we should add EOL Rubies in `matrix:ruby` above?

Reply:
Yep, that's my next task.

Reply:
Adding them one by one now...